docs/details/lapack.dox

```diff
@@ -252,6 +252,22 @@ I [3 3 1 1]
 
 \endcode
 
+=======================================================================
+
+\defgroup lapack_ops_func_pinv pinverse
+
+\ingroup lapack_ops_mat
+
+\brief Pseudo-invert a matrix
+
+This function calculates the Moore-Penrose pseudoinverse of a matrix \f$A\f$, using \ref af::svd at its core. If \f$A\f$ is of size **M x N**, then its pseudoinverse \f$A^+\f$ will be of size **N x M**.
+
+This calculation can be batched if the input array is three-dimensional (**M x N x P**). Each **M x N** slice along the third dimension will have its own pseudoinverse, for a total of **P** pseudoinverses in the output array (**N x M x P**).
```

Comment:
Instead of making the M x N x P statement bold perhaps use \f$M \times N \times P$\f. 

<a href="https://www.codecogs.com/eqnedit.php?latex=\inline&space;M&space;\times&space;N&space;\times&space;P" target="_blank"><img src="https://latex.codecogs.com/gif.latex?\inline&space;M&space;\times&space;N&space;\times&space;P" title="M \times N \times P" /></a>

Reply:
Yup I was on the fence about this one, since other functions' docs (like af::svd) just usually put bold. But it's probably better to be consistent within a doc page. I'll change this.